src/components/Home.js

```diff
@@ -50,17 +50,17 @@ export default function Home() {
               <aside className="menu">
                 <ul className="menu-list">
                   <li>
-                    <a href={''} className="text is-2">
+                    <a href={'http://'} className="text is-2">
```

Comment:
why `http` instead of nothing?

Reply:
because of annoying console warnings: `The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid`

error sadly also appears if href is just `#`. `http://` seems to suppress the warning. Because this links should are just placeholders for later, I thought that temporary fix should be fine